sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/BlobPartitionManager/BlobPartitionManagerLiveTests.cs

```diff
@@ -178,6 +217,50 @@ await using (StorageScope storageScope = await StorageScope.CreateAsync())
             }
         }
 
+        /// <summary>
+        ///    Verifies that returned eTag contains single quotes
+        /// </summary>
+        ///
```

Comment:
The summaries for the new tests are 1 blank space off.
```
/// <summary>
///   Verifies...
```
The first letter of `Verifies` should be placed directly below the letter `u`.